.github/workflows/cd.yml

```diff
@@ -52,12 +50,18 @@ jobs:
         id: vars
         run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
       - name: status
-        run: echo Build is tagged. Uploading artifact ${{ steps.vars.outputs.tag }} to Bintray.
+        run: echo Build is tagged. Uploading artifact ${{ steps.vars.outputs.tag }} to maven central.
       - name: Publish GitHub Pages
         run: ./gradlew --info -Dbuild.version="${{ steps.vars.outputs.tag }}" mkdocsPublish
-      - name: Upload Artifacts to Bintray
-        if: success() # only when previous step succeeds
-        run: ./gradlew --info  -Dbuild.version="${{ steps.vars.outputs.tag }}" bintrayUpload
+      - name: Set up Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Publish package
+        run: gradle -Dbuild.version="${{ steps.vars.outputs.tag }}" publish
```

Comment:
I think `setLibraryVersion`should be part of the publish step otherwise the version in the userAgent header will be missing ( also with the proper env variable called GITHUB_TAG)

Reply:
8194295d7